src/main/java/com/cloudbees/jenkins/support/impl/AboutJenkins.java

```diff
@@ -326,6 +326,7 @@ public String call() throws RuntimeException {
             result.append(min).append(" Name:    ").append(System.getProperty("java.vm.name")).append("\n");
             result.append(min).append(" Vendor:  ").append(System.getProperty("java.vm.vendor")).append("\n");
             result.append(min).append(" Version: ").append(System.getProperty("java.vm.version")).append("\n");
+            result.append(min).append(" Bit size: ").append(System.getProperty("sun.arch.data.model")).append("\n");
```

Comment:
Is this always guaranteed to be a property? What if `null` is returned? You could supply a default I think, if that helped.